lib/node_modules/@stdlib/array/base/cusome-by-right/docs/repl.txt

```diff
@@ -0,0 +1,83 @@
+{{alias}}( x, n, predicate[, thisArg] )
+    Cumulatively tests whether at least n array elements in a provided array
+    pass a test implemented by a predicate function,
+    while iterating from right-to-left.
+
+    The predicate function is provided three arguments:
+
+    - value: current array element.
+    - index: current array element index.
+    - arr: the input array.
+
+    Parameters
+    ----------
+    x: ArrayLikeObject
+        Input array.
+
+    n: number
+        Minimum number of successful elements.
+
+    predicate: Function
+        Predicate function.
+
+    thisArg: any (optional)
+        Execution context.
+
+    Returns
+    -------
+    out: Array
+        Output array.
+
+    Examples
+    --------
+    > function isPositive( v ) { return ( v > 0 ); };
+    > var x = [ 1, 1, 0, 0, 0 ];
+    > var y = {{alias}}( x, 2, isPositive )
```

Comment:
```suggestion
    > function fcn( v ) { return ( v > 0 ); };
    > var x = [ 1, 1, 0, 0, 0 ];
    > var y = {{alias}}( x, 2, fcn )
```

The alias `isPositive` is likely to conflict with built-in aliases.

cc @Planeshifter